os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -207,6 +223,11 @@ static void rtl8730e_control_backlight(uint8_t level)
 #if defined(CONFIG_LCD_ST7785) || defined(CONFIG_LCD_ST7701SN)
 	pwmout_write(&g_rtl8730e_config_dev_s.pwm_led, 1.0-pwm_level);
 #endif
+	if (g_rtl8730e_config_dev_s.pwm_level == 0 && level > 0) {
```

Comment:
What does this mean? Could you add some comment to explain?

Reply:
Updated